app/services/notifications/reactions/send.rb

```diff
@@ -54,11 +54,10 @@ def call
 
           json_data = reaction_json_data(recent_reaction, aggregated_reaction_siblings)
 
-          previous_siblings_size = 0
           notification = Notification.find_or_initialize_by(notification_params)
 
           old_json_data = notification.json_data
-          previous_siblings_size = old_json_data.dig("reaction", "aggregated_siblings")&.size if old_json_data
+          previous_siblings_size = old_json_data&.dig("reaction", "aggregated_siblings")&.size || 0
```

Comment:
nit: actually `old_json_data` is now redundant and can be removed

Reply:
ah ha - because we're not modifying notification.json_data until after we read this value and we don't use it again (that's why we have the previous value).

Good catch - thanks!